dev/io.openliberty.mcp/src/io/openliberty/mcp/annotations/WrapBusinessError.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) contributors to https://github.com/quarkiverse/quarkus-mcp-server
+ * Copyright (c) 2025 IBM Corporation and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Based on https://github.com/quarkiverse/quarkus-mcp-server/blob/main/core/runtime/src/main/java/io/quarkiverse/mcp/server/Tool.java
```

Comment:
Should this say WrapBusinessError instead of Tool?

Reply:
yes